src/content/en/updates/2019/02/using-twa.md

```diff
@@ -442,6 +442,119 @@ adb logcat | grep -e OriginVerifier -e digital_asset_links
 With the upload APK generated, you can now [upload the app to the Play
 Store](https://developer.android.com/studio/publish/upload-bundle).
 
+## Adding a Splash Screen {: #making-a-splash }
+
+Starting on **Chrome 75**, Trusted Web Activities are have support for Splash Screens.
```

Comment:
nit: remove "are"

Reply:
done

Reply:
Digital Marketing! | Work From Home! 
Make money online without any expenses!
Our Journey Since 24.12.2012
Subscribe & help us to build a blog !
http://www.mizansurovi1.blogspot.com
![1 banner](https://user-images.githubusercontent.com/6325672/59135234-04cfa300-89a0-11e9-9de1-808aa84b85af.png)

Reply:
![1 banner](https://user-images.githubusercontent.com/6325672/59135284-32b4e780-89a0-11e9-8a60-10e82311380d.png)
Digital Marketing! | Work From Home! 
Make money online without any expenses!
Our Journey Since 24.12.2012
Subscribe & help us to build a blog !
http://www.mizansurovi1.blogspot.com